torch/csrc/utils/python_numbers.h

```diff
@@ -18,6 +18,18 @@ inline PyObject* THPUtils_packInt64(int64_t value) {
   return PyLong_FromLongLong(value);
 }
 
+inline PyObject* THPUtils_packUnsignedInt64(uint64_t value) {
+#if PY_MAJOR_VERSION == 2
+  if (value <= INT32_MAX) {
+    return PyInt_FromLong(static_cast<long>(value));
+  }
+#endif
+  if (sizeof(unsigned long) == sizeof(uint64_t)) {
+    return PyLong_FromUnsignedLong(static_cast<unsigned long>(value));
+  }
```

Comment:
I don't think this is necessary. Just use `FromUnsignedLongLong`, which is guaranteed to be at least 64-bits